infra/bootstrapper/dev/locals.tf

```diff
@@ -0,0 +1,53 @@
+locals {
+  environment = {
+    prefix          = "dx"
+    location        = "italynorth"
+    location_short  = "itn"
+    env_short       = "d"
+    domain          = "playground"
+    app_name        = "playground"
+    instance_number = "01"
+  }
+
+  tags = {
+    CostCenter     = "TS000 - Tecnologia e Servizi"
+    CreatedBy      = "Terraform"
+    Owner          = "DevEx"
+    Environment    = "Dev"
+    Source         = "https://github.com/pagopa/dx-playground/blob/main/infra/bootstrapper/dev"
+    ManagementTeam = "Developer Experience"
+  }
+
+  core_state = {
+    resource_group_name  = "dx-d-itn-tfstate-rg-01"
+    storage_account_name = "dxditntfstatest01"
+    container_name       = "terraform-state"
+    key                  = "dx.core.dev.tfstate"
+  }
```

Comment:
suggestion: can we compute these values into bootstrap module and set them as defaults?